lib/Controller/RoomController.php

```diff
@@ -978,7 +978,7 @@ public function removeGuestFromRoom(string $token, string $participant): DataRes
 			return new DataResponse([], Http::STATUS_FORBIDDEN);
 		}
 
-		$room->removeParticipantBySession($targetParticipant);
+		$room->removeParticipantBySession($targetParticipant, 'remove');
```

Comment:
Please define constants for the reason strings in `Room.php`, e.g. `REASON_REMOVE` / `REASON_LEAVE` and use them instead of the hardcoded values.

Reply:
constants have been added